src/player.cpp

```diff
@@ -1902,6 +1902,16 @@ int player::run_cost( int base_cost, bool diag ) const
         }
     }
 
+    // Skates with only one wheel (roller shoes) are fairly less stable
+    // and fairly slower as well
+    if( worn_with_flag( "ROLLER_ONE" ) ) {
+        if( on_road ) {
+            movecost *= 0.85f;
+        } else {
+            movecost *= 1.55f;
```

Comment:
I suggest value less than 1.3 as in ROLLER_QUAD. 1.2 maybe.

Reply:
Ah, I was gonna just change it to 1.1 after it was pointed out to me that was really steep, thank you though!

Reply:
Glad I could help.